MPU/src/pedals.cpp

```diff
@@ -45,6 +45,16 @@ FaultStatus_t Pedals::readAccel()
 	//Calculate the correct amount of torque to command using precentage pressed value
 	appliedTorque = calcTorque(multiplier);
 
+	if (drive_state == PIT || drive_state == REVERSE) {
+		int16_t mph = (motorController->getMotorSpeed() * 0.013048225);
+		if (mph > 5) {
+			appliedTorque = 0;
+		}
+		else {
+			appliedTorque = appliedTorque * (5 - (4.5 + (mph / 10)));
```

Comment:
Dividing mph by 10 will always yield 0 given mph was defined as an int

Reply:
oof facts so ig we do have to make it a float

Reply:
Int could work, you'd just have to change your math here to multiply by 10 first or something along those lines

Reply:
yeah i could do like 50 - 45 + ... * 10 but float i think makes more sense anyways, it will be smoother than like set like increments